### ISSUE TITLE
Non 0 output for failed tests? (Usage in CI)

### ISSUE THREAD

**Issue by mog3n** [tnicola/cypress-parallel#28]
Hi there,

I am running cypress parallel in CI and was wondering if it's possible to return a non-zero output if one test fails?

Here's the command I'm running: `cypress-parallel -s cy:run -t 4 -d 'cypress/integration/v2'`

Thank you!

**Comment by jebadirad** [comment]
im facing the same issue. we are using this in CI and just discovered that we have some failing tests that is not causing the pipeline to fail

**Comment by mog3n** [comment]
@jebadirad I was able to create a python script that parses the results-

You want to output the cypress runner results to a file caled `results.txt`.

Then to parse the results, run this python script:

```
import sys
f = open('results.txt', 'r')

success = 0
fail = 0

for l in f:
    print(l)
    if "Results" in l:
        line = l.split()
        print(line)
        tests = line[6]
        success = line[8]
        fail = line[10]
        
        if tests == success == fail:
            tests = line[5]
            success = line[7]
            fail = line[9]

print("Tests", tests, "Succeeded:", success, "Failed:", fail)

if tests == success:
    # No failures
    sys.exit(0)
else:
    # There was at least one failure
    sys.exit(69)
```

**Comment by shrohitpatel** [comment]
Hello @jebadirad 

Thank you for adding support for a custom reporter. I am facing similar isssue with CI not failing the cypress-parallel task with custom reporter. Looks like the failing cypress task is skipped for custom reporter.

https://github.com/tnicola/cypress-parallel/pull/31/files#diff-347ff93ed2b00c93c817863e32fbac5b4fac71d7339a48378980e682777689f4R209

**Comment by tnicola** [comment]
Should be implemented in 0.6.0. Feel free to re-open the issue if not working.
